.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
```

Comment:
<!--
  6 failure:  " should be ' lin..., " should be ' lin..., " should be ' lin..., Unnecessarily quo..., Unnecessarily quo..., tab indentation e...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: .vscode/tasks.json;
  Line: 3;
-->
- :no_entry_sign: " should be ' line - tasks.json (quotemark)
- :no_entry_sign: " should be ' line - tasks.json (quotemark)
- :no_entry_sign: " should be ' line - tslint-with-errors.json (quotemark)
- :no_entry_sign: Unnecessarily quoted property 'semicolon' found. line - tslint-with-errors.json (object-literal-key-quotes)
- :no_entry_sign: Unnecessarily quoted property 'version' found. line - tasks.json (object-literal-key-quotes)
- :no_entry_sign: tab indentation expected line - tasks.json (indent)